frontend/src/scenes/experiments/RunningTimeCalculator/runningTimeCalculatorLogic.tsx

```diff
@@ -204,38 +279,96 @@ export const runningTimeCalculatorLogic = kea<runningTimeCalculatorLogicType>([
+                        Sample size formula:
+        
+                        N = (16 * variance) / d^2
+        
+                        Where:
+                        - `16` comes from statistical power analysis:
+                            - Based on a 95% confidence level (Z_alpha/2 = 1.96) and 80% power (Z_beta = 0.84),
+                              the combined squared Z-scores yield approximately 16.
+                        - `variance` is the estimated variance of the event count per user.
+                        - `d` is the absolute effect size (MDE * mean).
+                    */
+                    sampleSizeFormula = (16 * variance) / d ** 2
                 } else if (
                     metric.metric_type === ExperimentMetricType.MEAN &&
                     metric.metric_config.math === ExperimentMetricMathType.Sum
                 ) {
+                    /*
+                        Continuous property metric:
+                        - "mean" is the average value of the measured property per user (e.g., revenue per user).
+                        - MDE is applied as a percentage of this mean to compute `d`.
+        
+                        Formula:
+                        d = MDE * averagePropertyValuePerUser
+                    */
                     d = minimumDetectableEffectDecimal * averagePropertyValuePerUser
+
+                    /*
+                        Sample Size Formula for Continuous metrics:
+        
+                        N = (16 * variance) / d^2
+        
+                        Where:
+                        - `variance` is the estimated variance of the continuous property.
+                        - The formula is identical to the Count metric case.
+                    */
+                    sampleSizeFormula = (16 * variance) / d ** 2
+                } else if (metric.metric_type === ExperimentMetricType.FUNNEL) {
+                    /*
+                        Binomial metric (conversion rate):
+                        - Here, "mean" does not exist in the same way as for count/continuous metrics.
+                        - Instead, we use `p`, the baseline conversion rate (historical probability of success).
+                        - MDE is applied as an absolute percentage change to `p`.
+        
+                        Formula:
+                        d = MDE (absolute percentage point change)
+                    */
+                    d = minimumDetectableEffectDecimal
+
+                    /*
+                        Sample size formula:
+        
+                        N = (16 * p * (1 - p)) / d^2
+        
+                        Where:
+                        - `p` is the historical conversion rate (baseline success probability).
+                        - `d` is the absolute MDE (e.g., detecting a 5% increase means `d = 0.05`).
+                        - The variance is inherent in `p(1 - p)`, which represents binomial variance.
+                    */
```

Comment:
These comments are great, thanks for including them.

Reply:
+1